app/assets/stylesheets/professor.scss

```diff
@@ -6,58 +6,105 @@
   border-radius: 4px;
   margin-left:   0px;
   margin-right:  0px;
-  margin-top:    5px;
-  margin-bottom: 5px;
-  padding: 40px 40px;
-}
-
-#professor-header {
-	display: block;
-	width: 100%;
-	height: 100px;
-
-	#avg-rating, #avg-difficulty {
-		padding: 10px;		
-		margin-right: 50px;
-		height: 100px; 
-		width: 162px;
-		float: left; 
-		background-color: $light-blue; 
-		color: white; 
-		text-align: center; 
-	}
-
-	#professor-name {
-		padding: 10px;		
-	}
-
-	#professor-email { 
-		color: $dark-blue; 
-		text-align: center; 
-	}
-
-	#professor-email:hover {
-		color: $orange;
-	}
-
-	#professor-contact {
-		float: right;
-	}
-	
-}
-
-.subdepartment-name {
-	padding-left:10px;
-	// margin-bottom: 20px;
-	// text-align: center;
-}
-.professor-semester {
-	margin-bottom: 10px;
-}
-
-#professor-courses {
-	.course-panel {
-		margin-left: 10px;
-		margin-right: 10px;
-	}
-}
\ No newline at end of file
+  margin-top:    0px;
```

Comment:
Maybe we should put this in `app/javascript/components/professors/ProfessorPage.css`? Then we can import the styles in React.